src/main/resources/language/English.properties

```diff
@@ -413,14 +418,14 @@ ls.team.teamattitude_short.matchoftheseason=MOTS
 ls.team.trainingtype.setpieces=Set pieces
 ls.team.trainingtype.defending=Defending
 ls.team.trainingtype.scoring=Scoring
-ls.team.trainingtype.crossing=Crossing (Winger)
-ls.team.trainingtype.shooting=Shooting
-ls.team.trainingtype.shortpasses=Short passes
+ls.team.trainingtype.crossing=Winger
+ls.team.trainingtype.shooting=Scoring and Set Pieces
+ls.team.trainingtype.shortpasses=Passing
 ls.team.trainingtype.playmaking=Playmaking
-ls.team.trainingtype.goalkeeping=Goalkeeping
-ls.team.trainingtype.throughpasses=Through passes
-ls.team.trainingtype.defensivepositions=Defensive positions
-ls.team.trainingtype.wingattacks=Wing attacks
+ls.team.trainingtype.goalkeeping=Keeper
+ls.team.trainingtype.throughpasses=Passing (Defenders + Midfielders + Wingers)
+ls.team.trainingtype.defensivepositions=Defending (Defenders, Keepers + All Midfielders)
+ls.team.trainingtype.wingattacks=Winger (Wingers + Attackers)
```

Comment:
@tychobrailleur  This will fix the training type translations if poeditor will override the translations with this changes.